runtime/src/xcm_config.rs

```diff
@@ -510,6 +244,8 @@ impl xcm_executor::Config for XcmConfig {
     type HrmpChannelClosingHandler = ();
 }
 
+const FELLOWSHIP_ADMIN_INDEX: u32 = 1; // to be moved to some constants mod
```

Comment:
What is this ? Is this mandatory ?

Reply:
No it isn't, as we are not using any of these as of today. I also liked the idea of removing this, but then I thought to keep it consistent with the rest of the runtime, that is with the _governance_ part in this case.